app/Models/EntityEvent.php

```diff
@@ -403,12 +407,11 @@ public function mostRecentOccurrence(int $year, int $month, int $day, array $mon
         $reminderYear = $this->year;
         $reminderMonth = $this->month;
         $reminderDay = $this->day;
-
+        $overDate = false;
         // Recurring? We need to switch around the data a bit to figure out the most recent date
         if (!empty($this->is_recurring)) {
             if ($this->recurringMonthly()) {
                 //dump('monthly');
-                $reminderMonth = $month;
```

Comment:
Why remove this line? If it's recurring monthly but doesn't have more days, it will be the wrong value, no?